src/rgw/rgw_rados.cc

```diff
@@ -4000,7 +4000,14 @@ int RGWRados::open_gc_pool_ctx()
 
 int RGWRados::open_lc_pool_ctx()
 {
-  const char *lc_pool = get_zone_params().lc_pool.name.c_str();
+  const char *lc_pool;
+
+  if (!get_zone_params().lc_pool.name.empty()){
+    lc_pool = get_zone_params().lc_pool.name.c_str();
+  } else {
+    lc_pool = "default.rgw.lc";
```

Comment:
this is still problematic, we can have a name collision (after renaming a zone) you will need to use fix_zone_pool_name.
We also need to add the lc pool to new zones in RGWZoneParams::fix_pool_names.